git/gitgithubclient.py

```diff
@@ -40,7 +39,7 @@ def _commitData(self, commit):
 			"message": commit.commit.message
 		}
 
-	def commits(self, branch, since = 0, to = datetime.datetime.now() + datetime.timedelta(hours=24)):
+	def commits(self, branch, since=0, to=0):
```

Comment:
'To' value is still in timestamp format. As you can see for datetime I have added new 'to_dt' value.